pkg/pillar/zedUpload/azureutil/azure.go

```diff
@@ -161,9 +198,9 @@ func UploadAzureBlob(accountName, accountKey, containerName, remoteFile, localFi
 	file, _ := os.Open(localFile)
 	defer file.Close()
 	blob := container.GetBlobReference(remoteFile)
-	er := blob.CreateBlockBlobFromReader(file, nil)
-	if er != nil {
-		return er
+	putBlockErr := putBlockBlob(blob, file)
```

Comment:
Nit but the golang norm is to use the name "err" for the errors.
I'll pull this in its current state but please fix later to make the style match.